### ISSUE TITLE
Safe Search

### ISSUE THREAD

**Issue by Madbrad200** [bisoncorps/search-engine-parser#126]
Hello,

Is there any way to enable 'safe search' on google queries and if not, could that be added? Seems by default that it's turned off.

**Comment by deven96** [comment]
Yeah seems it is disabled by default... would entail setting the query string `safe=active`. Would you be interested in opening a PR for this @Madbrad200 ?
